### ISSUE TITLE
运行trainer时报错Error building extension 'fused_adam'

### ISSUE THREAD

**Issue by J-G-Y** [liucongg/ChatGLM-Finetuning#122]
如题

**Comment by J-G-Y** [comment]
前置报错Unsupported gpu architecture 'compute_80'

**Comment by liucongg** [comment]
with torch.cuda.amp.autocast(enabled=True, dtype=torch.bfloat16) as autocast, torch.backends.cuda.sdp_kernel(enable_flash=False) as disable:
    outputs = model(**batch, use_cache=False)
    loss = outputs.loss
    tr_loss += loss.item()
    model.backward(loss)
    torch.nn.utils.clip_grad_norm_(model.parameters(), 1.0)
    model.step()

**Comment by Stark-zheng** [comment]
> with torch.cuda.amp.autocast(enabled=True, dtype=torch.bfloat16) as autocast, torch.backends.cuda.sdp_kernel(enable_flash=False) as disable: outputs = model(**batch, use_cache=False) loss = outputs.loss tr_loss += loss.item() model.backward(loss) torch.nn.utils.clip_grad_norm_(model.parameters(), 1.0) model.step()

请问这段代码加在哪里？我也和楼主一样的错误，是在：
    model, optimizer, _, lr_scheduler = deepspeed.initialize(model=model, args=args, config=ds_config,
                                                             dist_init_required=True)
这里报的错误～

**Comment by liucongg** [comment]
听该是cuda版本的问题，cuda版本和装的要保持一致

**Comment by nothing7744** [comment]
同样遇到了这个问题
